client/vnsEPgDef_service.go

```diff
@@ -0,0 +1,233 @@
+package client
+
+import (
+	"fmt"
+
+	"github.com/ciscoecosystem/aci-go-client/container"
+	"github.com/ciscoecosystem/aci-go-client/models"
+)
+
+func (sm *ServiceManager) CreateEPgDef(name string, legvnode string, node_instance string, l4l7_service_graph_instance_scopeDn string, l4l7_service_graph_instance_graphDn string, l4l7_service_graph_instance_ctrctDn string, tenant string, description string, nameAlias string, vnsEPgDefAttr models.EPgDefAttributes) (*models.EPgDef, error) {
```

Comment:
scopeDn -> scope_dn , graphDn -> graph_dn and ctrctDn -> ctrct_dn

Reply:
also which DN of the few does this cover?

Reply:
I took care of this